signature-aggregator/README.md

```diff
@@ -0,0 +1,64 @@
+# Signature Aggregator
+
+This directory contains a lightweight stand-alone API for requesting signatures for a Warp message from Subnet validators.
+It is also used by `awm-relayer` for gathering signatures when configured to use AppRequest instead of the WarpSignature RPC client.
```

Comment:
nit: is WarpSignature meant to be one word

Reply:
Sort of since it was referring to a particular client but that's not it's proper name so split it up for now